### PR TITLE
[ROCM] fixing build brake and temporarily disable delayed kernel for rocm

### PR DESCRIPTION
**Pull request by pemeliya**
This PR fixes the ongoing build brake after https://github.com/openxla/xla/commit/28cfdcd27859e79cf10bd92779bc29f6404c79a0

and also I temporarily disable delayed kernel for ROCM since it significantly increases running times (to be investigated).

@xla-rotation: would you please have a look ?

### CODE REVIEW

**Review by beckerhe — APPROVED** [review]
Looks good to me. But note that the change with the delay kernel get rolled back, so we won't be able to merge this before we roll forward the change - which we intent to do, so this PR will still be needed.

**Comment by pemeliya** [comment]
ok I see, thanks for the info.. then let put this PR on hold until GpuTimer changes are merged

**Comment by akuegel** [comment]
This landed as part of the roll forward: https://github.com/openxla/xla/commit/a240f804899c6f334a45ffeaf6444c7d0d1af767#diff-a2b6eae7abf00c57b546d42c4317e1a37d5165e0b6c375e58444ad054a70b2ca

**Pull request closed without merging**
